### PR TITLE
43 feature adding capacity to event card

### PR DESCRIPTION
**Pull request by Tufiale**
Added tests to calendar page and also the feature of adding capacity to the event card

### CODE REVIEW

File: src/components/EventCard/EventCard.tsx [hunk 1]
```diff
@@ -77,6 +81,12 @@ const EventCard = ({
                 imageAltTag={`image for ${maker}`}
               />
             )}
+            <div className="event-card__capacity">
+              <p className="event-card__capacity-text">
+                There are {capacityMax - capacityCurrent} spaces left out of{" "}
+                {capacityMax}
+              </p>
+            </div>
           </div>
```
Comment (olirob93) [inline src/components/EventCard/EventCard.tsx:90]: this should be a conditional render for example. If the event is after todays date then it should say something like the attendance of this event is X or you can have how many spaces are available.

File: src/pages/Calendar/Calendar.tsx [hunk 2]
```diff
@@ -108,6 +108,8 @@ const CalendarPage = ({ eventData }: CalendarPageProps) => {
                   date={event.date}
                   textContent={event.description}
                   galleryArray={event.images}
+                  capacityCurrent={0}
+                  capacityMax={0}
```
Comment (olirob93) [inline src/pages/Calendar/Calendar.tsx:112]: I can see why you have done this to make it work with historic events but see my comment above as a way to fix this so we can pass the actual data coming from the DB

**Review by olirob93 — CHANGES_REQUESTED** [review]
A couple of changes

File: src/components/EventCard/EventCard.tsx (hunk 1 above)
Comment (Tufiale) [reply src/components/EventCard/EventCard.tsx]: Done!

**Review by Charlie-robin — APPROVED** [review]
Looking good

**Review by olirob93 — APPROVED** [review]

**Pull request merged**